packages/verifiable-credentials/package.json

```diff
@@ -31,6 +31,7 @@
         "@ethersproject/wallet": "5.7.0",
         "@krebitdao/eip712-vc": "1.0.3",
         "date-fns": "2.30.0",
+        "node-fetch": "^3.3.2",
```

Comment:
u shouldnt need node-fetch

node 18 includes fetch, node 16 does not (but it is end of life and should not be used anymore). so we should be using node 18